app/admin/sponsor.rb

```diff
@@ -32,7 +32,7 @@
     column :request_fulfilled
     column :sponsor_type
     column :country do |_sponsor|
-      "(#{ISO3166::Country[_sponsor.country]}) #{t _sponsor.country, locale: :ar}"
+      "(#{ISO3166::Country[_sponsor.country]}) #{t _sponsor.country, locale: :ar, default: ''}"
```

Comment:
http://guides.rubyonrails.org/i18n.html, Section 4.1.2 says we can use a dictionary value as a default which would allow us to only change the message in one place if it had to change.

Quote from above:

If the :default value is a Symbol, it will be used as a key and translated. One can provide multiple values as default. The first one that results in a value will be returned.

E.g., the following first tries to translate the key :missing and then the key :also_missing. As both do not yield a result, the string "Not here" will be returned:

```
I18n.t :missing, default: [:also_missing, 'Not here']
# => 'Not here'
```

Thoughts?